### ISSUE TITLE
Broken group resolving

### ISSUE THREAD

**Issue by ondratra** [mrmlnc/fast-glob#263]
Group resolving is broken in the newest version `3.2.2`. The feature was working well in older versions (at least in `3.1.1`). Issue #257 was probably not fixed.

### Actual behavior
No files found when using `somePath/(option1|option2)/*.extension`.

### Expected behavior
Files found.

### Steps to reproduce
```
const glob = require('fast-glob')

async function test() {
  const files1 = await glob([
    __dirname + '/testContent/(subfolder1|subfolder2)/*.js'
  ])
  const files2 = await glob([
    __dirname + '/testContent/subfolder1/*.js',
    __dirname + '/testContent/subfolder2/*.js',
  ])

  console.log(files1, files2)
}

test()
```
Assuming that some files exist in relevant paths, variables `files1` and `files2` should be the same, but they are not.

### Code sample
Sandbox with the problem can be found here https://repl.it/@ondratra/fast-glob-bug .

**Comment by mrmlnc** [comment]
After `scan` the last part of the pattern has forward slash:

```
[..., '(subfolder1|subfolder2)/*.js']
```

**Comment by mrmlnc** [comment]
Blocked by https://github.com/micromatch/picomatch/pull/65

**Comment by mrmlnc** [comment]
Fixed by [`picomatch@2.2.2`](https://github.com/micromatch/picomatch/blob/master/CHANGELOG.md#222-2020-03-21).

In the simplest case, run this command to get the latest versions of dependencies:

```
npm i fast-glob
```

**Comment by ondratra** [comment]
It just noticed there is no package-lock.json in the project. You should consider adding it for better dependency management.
